x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.js

```diff
@@ -211,9 +208,7 @@ class LinksMenuUI extends Component {
         },
       },
     });
-    await navigateToApp(PLUGIN_ID, {
-      path: singleMetricViewerLink,
-    });
+    window.open(singleMetricViewerLink, '_blank');
```

Comment:
shall we use `MlHref` instead of `window.open`?

Reply:
We'll keep this one here for now for 7.10.0.